prettier/tests/__snapshots__/run_tests.js.snap

```diff
@@ -4103,31 +4129,51 @@ This is a simple dialog rendered by using a macro and a call block.
     use_page_url=True,
     overrideable=True %}
     {% widget_attribute "pinterest"
-      is_json=True %}{"custom_link_format": "", "pinterest_media":
-      "http://cdn2.hubspot.net/hub/158015/305390_10100548508246879_837195_59275782_6882128_n.jpg",
-      "enabled": true, "network": "pinterest", "img_src":
-      "https://static.hubspot.com/final/img/common/icons/social/pinterest-24x24.png"}
-  {% end_widget_attribute %}
+      is_json=True %}
+    {
+      "custom_link_format": "",
+      "pinterest_media": "http://cdn2.hubspot.net/hub/158015/305390_10100548508246879_837195_59275782_6882128_n.jpg",
+      "enabled": true,
+      "network": "pinterest",
+      "img_src": "https://static.hubspot.com/final/img/common/icons/social/pinterest-24x24.png"
+    }
+    {% end_widget_attribute %}
     {% widget_attribute "twitter"
-      is_json=True %}{"custom_link_format": "", "enabled": true, "network":
-      "twitter", "img_src":
-      "https://static.hubspot.com/final/img/common/icons/social/twitter-24x24.png"}
-  {% end_widget_attribute %}
+      is_json=True %}
+    {
+      "custom_link_format": "",
+      "enabled": true,
+      "network": "twitter",
+      "img_src": "https://static.hubspot.com/final/img/common/icons/social/twitter-24x24.png"
+    }
+    {% end_widget_attribute %}
     {% widget_attribute "linkedin"
-      is_json=True %}{"custom_link_format": "", "enabled": true, "network":
-      "linkedin", "img_src":
-      "https://static.hubspot.com/final/img/common/icons/social/linkedin-24x24.png"}
-  {% end_widget_attribute %}
+      is_json=True %}
+    {
+      "custom_link_format": "",
+      "enabled": true,
+      "network": "linkedin",
+      "img_src": "https://static.hubspot.com/final/img/common/icons/social/linkedin-24x24.png"
+    }
+    {% end_widget_attribute %}
     {% widget_attribute "facebook"
-      is_json=True %}{"custom_link_format": "", "enabled": true, "network":
-      "facebook", "img_src":
-      "https://static.hubspot.com/final/img/common/icons/social/facebook-24x24.png"}
-  {% end_widget_attribute %}
+      is_json=True %}
+    {
+      "custom_link_format": "",
+      "enabled": true,
+      "network": "facebook",
+      "img_src": "https://static.hubspot.com/final/img/common/icons/social/facebook-24x24.png"
+    }
+    {% end_widget_attribute %}
     {% widget_attribute "email"
-      is_json=True %}{"custom_link_format": "", "enabled": true, "network":
-      "email", "img_src":
-      "https://static.hubspot.com/final/img/common/icons/social/email-24x24.png"}
-  {% end_widget_attribute %}
+      is_json=True %}
+    {
+      "custom_link_format": "",
+      "enabled": true,
+      "network": "email",
+      "img_src": "https://static.hubspot.com/final/img/common/icons/social/email-24x24.png"
+    }
+    {% end_widget_attribute %}
```

Comment:
This is a snapshot test. Nice to see the diff here and it's also helps to catch issues during dev quickly.